genesis/parsing/accountsParser.go

```diff
@@ -362,9 +362,35 @@ func createMiniBlocks(shardIDs []uint32, blockType block.Type) []*block.MiniBloc
 	return miniBlocks
 }
 
+func (ap *accountsParser) setMintingTxsPoolAndMiniBlock(
```

Comment:
maybe you can change the name of the function 
`putMintingTxsInPoolAndCreateMiniblock`

Reply:
:+1: